### ISSUE TITLE
Create new MondooOperatorConfig CRD

### ISSUE THREAD

**Issue by joelddiaz** [mondoohq/mondoo-operator#193]
New MondooOperatorConfig CRD can be used to control behavior for the mondoo-operator itself that is separate from the scanning/config details in the MondooAuditConfig CRD.

This will allow setting up things like Metrics enabling/disabling where it doesn't make sense to setup Metrics for each MondooAuditConfig when the metrics are really related to the mondoo-operator and not the scannning configuration.

We can also store global behavior settings like telling the mondoo-operator to bypass the image:tag resolution to image:SHA (to give workarounds if DockerHub or whatever other registry is being pulled from happens to be down).

The CRD Spec would look like:

```
type MondooOperatorConfigSpec struct {
        Metrics Metrics `json:"metrics,omitempty"`
        SkipContainerResolution bool `json:"skipContainerResolution,omitempty"`
}
```

And in the future, settings/configuration that are specific to the mondoo-operator can be implemented here and kept separate from the MondooAuditConfig.Spec.

**Comment by joelddiaz** [comment]
Addressed in most recent commits to  #186

**Comment by chris-rock** [comment]
added in #186
